nx/guides/intro-to-nx.livemd

```diff
@@ -264,7 +264,7 @@ suggest that they would work on primitive values, called scalars.
 Indeed, a tensor can be a scalar:
 
 ```elixir
-pi = Nx.tensor(3.1415, type: :f32)
+pi = Nx.Constants.pi()
```

Comment:
I think in this case we want to show how to build it by hand, so we should likely revert. :)

Reply:
```suggestion
pi = Nx.tensor(3.1415, type: :f32)
```